File: DS Assignments/ProgrammingAssignment1/ProgrammingAssignment1/src/programmingassignment1/Janitor.java
// COSC 2320 Summer 2015
// Ali Elsaadi
// Progamming Assignment 1
// This is my own work; I will not post
package programmingassignment1;



public class Janitor extends Adminstrator{
    private boolean sweeping;
    public Janitor ()
    {
        super (" ", 0, " ");
        sweeping = false;
    }
    public Janitor (String empName, int empNum, String dept, boolean sw)
    {
        super(empName, empNum, dept);
        sweeping = sw;
    }
    public void setIsSweeping (boolean isS)
    {
        sweeping = isS;
        
    }
    public boolean getIsSweeping()
    {
        return sweeping;
    }
    public String toString()
    {
        char SW = 'N';
        if (sweeping)
            SW='T';
        return super.toString() + "Sweeping: " + SW;
    }
    
}

File: DS Assignments/ProgrammingAssignment4/ProgrammingAssignment4/ProgrammingAssignment4/Janitor.cpp
#include "Janitor.h"

Janitor::Janitor()
{
	Janitor::Adminstrator( " " , 0 , " " );
	sweeping = false;
}
Janitor::Janitor(string empName, int empNumber, string dept, bool sw)
{
	Janitor::Adminstrator(empName, empNumber, dept);
	sweeping = sw;
}
void Janitor :: setIsSweeping(bool isS)
{
	sweeping = isS;
}
bool Janitor :: getIsSweeping()
{
	return sweeping;
}
string Janitor::toString()
{
	char SW = 'N';
	if (sweeping)
		SW = 'T';
	return Janitor::Adminstrator::toString() + SW;
}